games-puzzle/nudoku/nudoku-0.2.5.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+inherit autotools
+
+SLOT="0"
+
+DESCRIPTION="ncurses based sudoku game."
+HOMEPAGE="http://jubalh.github.io/nudoku"
+SRC_URI="https://github.com/jubalh/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="GPL-3"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="sys-libs/ncurses:="
```

Comment:
ncurses has two slots, of which slot '5' seems to be a legacy slot. Your package properly links against 6.0-r2:0 on my system so this should *probably* rather specify `:0=` here.